generate-sample-cert.sh

```diff
@@ -1,6 +1,4 @@
 #!/bin/sh
 openssl genrsa 1024 > sample.key
-openssl req -new -x509 -nodes -sha1 -days 3650 -key sample.key > sample.cert
-cat sample.cert sample.key > sample.pem
-rm sample.key sample.cert
-
+openssl req -new -x509 -nodes -sha1 -days 3650 -key sample.key > sample.crt
+cat sample.crt sample.key > sample.pem
```

Comment:
Yes, sha256 definitely.
You're right, the .pem can be removed if we use "--ca-certificate sample.crt" instead of "--ca-certificate sample.pem" with wget. I can change the commands in the README.